workshop/templates/workshop/index_2023.html

```diff
@@ -98,8 +98,11 @@ <h5 class="m2-txt4 p-b-25">
           {{workshop.location}}
         </p>
 
-        
-        <small class="m2-txt1 p-b-25"> <b class="m2-txt2">Workshop</b> has been started <br/> Don't miss it!</small>
+        <small class="m2-txt1 p-b-25"> <b class="m2-txt2">Latest workshop has been announced! <br/> 
+          <a href="https://workshop.dipy.org" style="color: white; text-decoration: underline">Click here to register </a>
+          </b>
```

Comment:
I am leading them to the latest workshop page. Because leading them directly to the newest registration will lead to missing the information about the latest workshop

If we want to move to the registration section, the link must be particular to the workshop year that needs to be updated yearly.